src/rpc/rawtransaction.cpp

```diff
@@ -149,6 +149,7 @@ static std::vector<RPCArg> CreateTxDoc()
         },
         {"outputs", RPCArg::Type::ARR, RPCArg::Optional::NO, "The outputs (key-value pairs), where none of the keys are duplicated.\n"
                 "That is, each address can only appear once and there can only be one 'data' object.\n"
+                "Either {\"data\"} or {\"address\"}, are required, but specifying both is optional.\n"
```

Comment:
I find this wording confusing. Do you mean to say that the user can combine data and address entries? Or that you can provide data or an address directly without putting "data" or "address" in front of it?

Examples can be helpful. Maybe say "see send RPC for examples" and then expand the `send` RPC docs with said examples.

Reply:
> Do you mean to say that the user can combine data and address entries? 

The outputs array, `outputs []`, must contain at minimum either an obj containing `{"address" ".."}` or an obj containing `{"data": ".."}`. You can also specify both, but you must have at least one.

I'm reading over this block of help text again. 
```
(json array, required) The outputs (key-value pairs), where none of the keys are duplicated.
                                      That is, each address can only appear once and there can only be one 'data' object.
                                      For compatibility reasons, a dictionary, which holds the key-value pairs directly, is also
                                      accepted as second parameter.
```

just using separate objs
```json 
[
    {
        "bc1address1":"0.01",
    },
    {
        "bc1address2":"0.02",
    },
    {
        "data":"ff00ff00ff"
    }
]
```
or...
Also note:
`a dictionary, which holds the key-value pairs directly, is also accepted as second parameter.`
I believe this is saying you can also do something like:
```json 
[
    {
        "bc1address1":"0.01",
        "bc1address2":"0.02",
        "data":"ff00ff00ff"
    }
]
```
I think this style is for compatibility reasons per docs, so I'm guessing it's preferred/future proof to break everything out into individual objects. Please correct me if I'm wrong!